requirements.txt

```diff
@@ -5,6 +5,7 @@ autopep8>=1.3
 fire
 google.cloud.logging
 grpcio-tools
+keras
```

Comment:
what's this about?

Reply:
We build the tensorflow install package in order to extract the C++ headers and keras is a required TF dependency now :/

I _really_ want to start using the user's TF install.